test/sshkit/ssh_test.exs

```diff
@@ -155,48 +126,36 @@ defmodule SSHKit.SSHTest do
   end
 
   describe "run/3" do
-    @options [ssh_modules: %{ssh: SSHSandboxSuccess, ssh_connection: :ssh_connection}]
-    test "error if Channel cannot be opened" do
-      conn = %SSHKit.SSH.Connection{
+    @tag ssh_connection: :error
+    test "error if Channel cannot be opened", %{ssh_modules: ssh_modules} do
+      conn    = %SSHKit.SSH.Connection{
         host:        'test',
         options:     [user_interaction: false],
         port:        22,
         ref:         :sandbox,
-        ssh_modules: Keyword.get(@options, :ssh_modules)
+        ssh_modules: ssh_modules
       }
-      assert run(conn, "uptime") == {:error, :closed}
-    end
 
-    @options [ssh_modules: %{ssh: SSHSandboxSuccess, ssh_connection: SSHSandboxConnectionSuccess}]
-    test "sucessfully execute command on connection and return result" do
```

Comment:
you remove the test to a successful connection. is there a reason for it?

Reply:
🤔  I cannot really remember tbh... Looking at it now, I think it was probably only an accident 🙈 

I'm going to add it back!

Reply:
Ok, it's back now ✌️

Reply:
Ok, getting a weird behavior of the test on Travis... Will investigate and fix

Reply:
Ah, now I remember: running a command and not being able to control the remote side will make the channel listen for messages forever and / or timeout. I could not yet come up with a sane way of testing this - would be happy for suggestions though 🙏